src/entities.rs

```diff
@@ -25,22 +27,26 @@ impl Entity {
     ///
     /// No particular structure is guaranteed for the returned bits.
     ///
+    /// If bits does not correspond to a possible `NonZeroU64` value returns `None`
+    ///
```

Comment:
This text is confusing, as there is no user-controlled value named "bits", and the function cannot possibly return `None`.